dvc/config.py

```diff
@@ -31,6 +31,18 @@ def __init__(self, msg, cause=None):
         )
 
 
+class NoRemoteRepositoryError(ConfigError):
+    def __init__(self, command, cause=None):
+        msg = (
+            "No remote repository specified. Setup default repository with\n"
```

Comment:
Do we really call remotes remote repositories? This sounds ambiguous with git and dvc repo.